### ISSUE TITLE
Can not work well for Medical Image Segmentation with conv3d Model

### ISSUE THREAD

**Issue by Itsanewday** [LiheYoung/UniMatch#58]
Great work!
Recently, i adopt this unimatch idea in the medical image segmentation on the semi-supervised abdominal organs segmentation task. I find that it not works for small organs and tumor. Can you help me?

**Comment by LiheYoung** [comment]
Hi,

The current information is limited. You need to provide your experimental details, such as the number of labeled cases, semi-supervised training logs, and the fully-supervised performance.

**Comment by Itsanewday** [comment]
Thanks for your reply! 
I have about 200 labeled and 800 unlabeled CTs. When trained with gt and pseudo labels the mean Organ_DSC is 0.9078, Tumor_DSC is about 0.3; while trained with unimatch, the mean Organ_DSC is 0.8842, Tumor_DSC is 0.0910 which is really bad. 
The major difference i make in the code is that i use the gradient accumulation instead of a large batch_size because of the GPU Memory limit. And i didn't use the mixmatch in augmentation. Did you test unimatch on 3d model for medical image?

**Comment by LiheYoung** [comment]
I have not explored our UniMatch for 3D medical images. But you may try different confidence thresholds and different strong augmentations (e.g., remove the color transformations, and add CutMix).

**Comment by Itsanewday** [comment]
Thanks!

**Comment by Itsanewday** [comment]
> I have not explored our UniMatch for 3D medical images. But you may try different confidence thresholds and different strong augmentations (e.g., remove the color transformations, and add CutMix).

Thanks a lot for your kindly reply! I add cut-mix and retrain the network with more epochs, now the UniMatch has slightly 
 lower performance as the full supervised methods! Thank you again!
